README.md

```diff
@@ -197,11 +197,240 @@ To check that formatting is valid without actually making changes, run `npm run
 1. Install and configure GitHub CLI: https://docs.github.com/en/github-cli/github-cli/quickstart
 2. Choose which documentation you want to regenerate: `qiskit`, `qiskit-ibm-provider`, or `qiskit-ibm-runtime`
 3. Determine the current version of the published stable documentation, e.g. at https://github.com/Qiskit/qiskit/releases
-4. Find a link to a CI artifact with the project's documentation, e.g at https://github.com/Qiskit/qiskit/suites/17881600359/artifacts/1026798160. To find this, go to the "Summary" page in GitHub Actions for the CI run you want to build, then scroll down to "Artifacts".
+4. Find a link to a CI artifact with the project's documentation, e.g at https://github.com/Qiskit/qiskit/suites/17881600359/artifacts/1026798160. To find this, first find the CI run for the commit you are trying to build docs for. This should come from a stable commit, not from a Pull Request. Then, go to the "Summary" page in GitHub Actions for the CI run. Scroll down to "Artifacts" and look for the artifact related to documentation, such as `html_docs`. Copy the link by right-clicking on the artifact.
```

Comment:
CC @kt474  - this is in response to the confusion with generating Runtime docs. Feedback welcomed!